aiohttp/client_reqrep.py

```diff
@@ -699,6 +699,7 @@ def __init__(
 
         self.method = method
         self.cookies: SimpleCookie[str] = SimpleCookie()
+        self.certificate: Optional[Dict[str, Union[Tuple[Any, ...], int, str]]] = None
```

Comment:
The type appears to be wrong. Typeshed defines it as: https://github.com/python/typeshed/blob/9d6fc1dafdf9d27a9ea87e210ff6e287d00a61d5/stdlib/ssl.pyi#L11

We could probably do even better and define a TypedDict if we wanted to.